src/cmd/aem-upload.js

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import chalk from 'chalk';
+import fs from 'fs';
+import inquirer from 'inquirer';
+import uploadPackageToAEM from '../aem-uploader/package/upload-package-to-aem.js';
```

Comment:
Does this module name need to be so descriptive? Since the module is alreayd in an `aem` folder would this be acceptable?

```
../aem/uploadPackage.js
```